### PR TITLE
Pass1-345: erase external flash after signing

### PR DESCRIPTION
**Pull request by mjg-foundation**

### CODE REVIEW

**Comment by mjg-foundation** [comment]
@FoundationKen what do you think about getting this ino 2.0.5 for testing?

**Comment by mjg-foundation** [comment]
@FoundationKen did you have a comment on this somewhere else in linear/twist? I remember reading it but can't find it now.

File: ports/stm32/boards/Passport/modules/utils.py [hunk 1]
```diff
@@ -1240,12 +1240,10 @@ def set_screen_brightness(value):
...
 
-async def clear_psbt_flash(psbt_len):
+async def clear_psbt_flash():
     from utils import spinner_task
     from tasks import clear_psbt_from_external_flash_task
-    from public_constants import TXN_INPUT_OFFSET
 
-    await spinner_task('Clearing transaction from flash', clear_psbt_from_external_flash_task,
-                       args=[None, psbt_len, TXN_INPUT_OFFSET])
+    await spinner_task('Clearing transaction from flash', clear_psbt_from_external_flash_task)
```
Comment (jeandudey) [inline ports/stm32/boards/Passport/modules/utils.py:1247]: I'm not sure we should indicate that we are clearing the PSBT from the flash, maybe a "Cleaning" or variations.

**Review by jeandudey — CHANGES_REQUESTED** [review]
LGTM, just the nit pick about the copy on the spinner task.

Tested and works correctly.

File: ports/stm32/boards/Passport/modules/tasks/copy_psbt_file_to_external_flash_task.py [hunk 2]
```diff
@@ -34,6 +34,10 @@ async def copy_psbt_file_to_external_flash_task(on_done, on_progress, filename,
             psbt_len = fd.seek(0, 2)
             fd.seek(0)
 
+            if psbt_len > MAX_TXN_LEN:
+                await on_done(0, None, Error.PSBT_INVALID)
```
Comment (FoundationKen) [inline ports/stm32/boards/Passport/modules/tasks/copy_psbt_file_to_external_flash_task.py:38]: We should differentiate between invalid PSBTs and ones that are just too long for us.  Add another error like `Error.PSBT_TOO_LARGE` and then show a different error in the code that calls this task.

File: ports/stm32/boards/Passport/modules/tasks/copy_psbt_to_external_flash_task.py [hunk 3]
```diff
@@ -26,6 +26,10 @@ async def copy_psbt_to_external_flash_task(on_done, on_progress, data, offset):
         # See how long it is -- This version of seek returns the final offset
         psbt_len = len(data)
 
+        if psbt_len > MAX_TXN_LEN:
```
Comment (FoundationKen) [inline ports/stm32/boards/Passport/modules/tasks/copy_psbt_to_external_flash_task.py:29]: Same comment as above for `Error.PSBT_TOO_LARGE`.

**Review by FoundationKen — APPROVED** [review]

**Review by FoundationKen — APPROVED** [review]

**Pull request merged**